### ISSUE TITLE
Eliminate language of "building blocks"

### ISSUE THREAD

**Issue by jpmckinney** [open-contracting/standard#1347]
The new Primer explains the concept of the "schema" – a term we use frequently. "Building blocks" is not very helpful, because:

1. It is a metaphor for an abstract concept (sub-schema). Using familiar words in a metaphorical sense to describe an abstract concept just adds another layer of abstraction and is harder to understand.
2. Having learned the term "schema", the term "sub-schema" is much easier to understand, as it expresses the relationship between the two concepts (e.g. the Organization sub-schema is part of the Release schema).
3. Publishers do not "build" OCDS data like LEGO blocks. The mapping step is more like using a lookup table and documenting relationships. If we were to retain a metaphor, "build" is not the right one.

Thankfully, there are only about 14 occurrences in the current documentation, so this should be quick. In some contexts, "object" is a more appropriate replacement (where it refers to the data representation, not the schema representation).

**Comment by JachymHercher** [comment]
Great news, at least for me! "Blocks" is one of the concepts that I am still confused by. That being said, out of curiosity, what are/were "blocks"? The best guesses I came up with are: 
1) Everything exactly one layer under `definitions` in the schema. If it is about nothing but location in the schema, then why is it important?  
2) Everything referred to from multiple places in the schema. However, that doesn't hold - e.g. `Organization` is just referred to from `parties`. (This guess sounds like "sub-schema"?)
3) Any sub-schema that contains fields which must be used together. However, that doesn't hold - e.g. `country` could be useful outside of `Address`. (E.g. eForms have something like "In cases of joint cross-border procurement, the law of the following country applies".)

____

(2. and 3. are great arguments. I'm not sure about 1. - I've often heard metaphors mentioned as crucial tools for understanding abstract concepts...)

**Comment by JachymHercher** [comment]
Ah, https://ocds-standard-development-handbook.readthedocs.io/en/latest/meta/schema_style_guide.html#definitions explains it. I understand it as essentially option 2. (just "could be referred to", not "is referred to"); with 3. always being a process (see e.g. https://github.com/open-contracting/standard/issues/893). In any case, "sub-schema" is way better.

**Comment by jpmckinney** [comment]
Yes, "building block" is typically used to refer to your (2), but probably not consistently.

I agree that metaphors can help; "building block" is just not a good metaphor.

**Comment by jpmckinney** [comment]
Moving to 1.2 as occurs on normative pages.

**Comment by odscjen** [comment]
Noting that this is probably the place to also remove instances of the simpler "block" as well as per the Common Conventions section of the style guide
